LaTeX/DocStrip.sublime-syntax

```diff
@@ -0,0 +1,96 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://www.texlive.info/CTAN/macros/latex/base/docstrip.pdf
+name: TeX (DocStrip)
+scope: source.tex.docstrip
+version: 1
+extends: Packages/LaTeX/TeX.sublime-syntax
+
+file_extensions:
+  - ins
```

Comment:
```suggestion
file_extensions: []
```
https://fileinfo.com/extension/ins

Reply:
I'm not really getting the feeling that any of the other usages of the "ins" extensions would be more likely than this one tbh.

> [9 file types](https://fileinfo.com/extension/ins) use the .ins file extension.
> 
> 1. [LaTeX Installer Script](https://fileinfo.com/extension/ins#latex_installer_script)
> 2. [Ensoniq Instrument](https://fileinfo.com/extension/ins#ensoniq_instrument)
> 3. [Internet Naming Service File](https://fileinfo.com/extension/ins#internet_naming_service_file)
> 4. [Adlib Tracker Instrument File](https://fileinfo.com/extension/ins#adlib_tracker_instrument_file)
> 5. [Internet Settings File](https://fileinfo.com/extension/ins#internet_settings_file)
> 6. [Sample Cell II Instrument Definition File](https://fileinfo.com/extension/ins#sample_cell_ii_instrument_definition_file)
> 7. [Inspiration File](https://fileinfo.com/extension/ins#inspiration_file)
> 8. [AGT Game Instruction File](https://fileinfo.com/extension/ins#agt_game_instruction_file)
> 9. [Internet Explorer Installation Settings](https://fileinfo.com/extension/ins#internet_explorer_installation_settings)

Reply:
Ok, than leave it as is.